spec.bs

```diff
@@ -544,6 +591,11 @@ The [=steps to respond to a payment request=] for this payment method, for a giv
     "{{PublicKeyCredentialRequestOptions/userVerification}}" → {{UserVerificationRequirement/required}},
     "{{PublicKeyCredentialRequestOptions/extensions}}" → |extensions|]».
 
+  **TODO**: Should the browser agent not extend the 'challenge' object passed in by putting it inside a new JSON structure whihc includes the payment amount and payee details? 
```

Comment:
No, that is the purpose of the extension to the CollectedClientData, to explicitly avoid hacking the `challenge`: https://w3c.github.io/secure-payment-confirmation/#sctn-collectedclientpaymentdata-dictionary